### PR TITLE
Release v2.1.9

### PR DESCRIPTION
**Pull request by kozlovic**
This is based on branch_2_1_x and includes the following fixes:

- Possible panic if server receives a maliciously crafted JWT.
- User and claims activation revocation checks.
- Panic on shutdown while accepting TLS client connections.
- Added defensive code for handling of Leafnode connections.

Signed-off-by: Ivan Kozlovic <ivan@synadia.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ If you are interested in contributing to NATS, read about our...
 [License-Image]: https://img.shields.io/badge/License-Apache2-blue.svg
 [Fossa-Url]: https://app.fossa.io/projects/git%2Bgithub.com%2Fnats-io%2Fgnatsd?ref=badge_shield
 [Fossa-Image]: https://app.fossa.io/api/projects/git%2Bgithub.com%2Fnats-io%2Fgnatsd.svg?type=shield
-[Build-Status-Url]: https://travis-ci.org/nats-io/nats-server
-[Build-Status-Image]: https://travis-ci.org/nats-io/nats-server.svg?branch=master
-[Release-Url]: https://github.com/nats-io/nats-server/releases/tag/v2.1.8
-[Release-image]: https://img.shields.io/badge/release-v2.1.8-1eb0fc.svg
+[Build-Status-Url]: https://travis-ci.com/github/nats-io/nats-server
+[Build-Status-Image]: https://travis-ci.com/nats-io/nats-server.svg?branch=master
```
Comment (derekcollison) [inline README.md:31]: Should this just be for this branch and not master?

Comment (kozlovic) [reply README.md:31]: Yes, but anybody can technically select the branch in the UI and will have whatever is displayed in the README. So why not update there too.

Comment (kozlovic) [reply README.md:31]: Oh, I see what you mean..

Comment (kozlovic) [reply README.md:31]: will update all references from master to `branch_2_1_x` to reflect actual values.

Comment (kozlovic) [reply README.md]: I pushed the update.

**Review by derekcollison — APPROVED** [review]
LGTM

**Review by philpennock — APPROVED** [review]

**Pull request merged**